README.md

```diff
@@ -217,3 +220,9 @@ config :torch,
   i18n_backend: MyApp.CustomMessagesBackend
   template_format: "eex" || "slime"
 ```
+
+## Copyright and License
+
+Copyright (c) 2016 Infinite Red, Inc.
+
+This software is released under the [MIT License](./LICENSE.md).
```

Comment:
This whole added section is not needed (somewhat incorrect) and can/should be removed.

Reply:
You're right about the incorrectness, it supposed to be:

```
Original work Copyright (c) 2016 Infinite Red, Inc.                                                     
Modifed work Copyright (c) 2019 MojoTech, LLC. 
```

I've removed it as requested.